test/writer.test.js

```diff
@@ -0,0 +1,248 @@
+'use strict';
+
+/* global test, expect, beforeEach, jest */
```

Comment:
you can do `env: { jest: true }` in eslintrc (my other comment about the same thing got lost in the eternal void of the internets)

Reply:
Did not know this, cheers. However, the only thing i've found in the past is that sometimes editor plugins don't always respect these sorts of global overrides when not in the same file thats being linted (ie. in package.json or wherever). Other than that i'd prefer not to have to litter the tops of files with these sorts of comments so i'll experiment and see how it goes.